src/StringViews.jl

```diff
@@ -86,6 +86,7 @@ Base.:(==)(s1::StringViewAndSub, s2::StringAndSub) = s2 == s1
 
 Base.typemin(::Type{StringView{Vector{UInt8}}}) = StringView(Vector{UInt8}(undef,0))
 Base.typemin(::T) where {T<:StringView} = typemin(T)
+Base.one(::T) where {T<:StringView} = typemin(T)
```

Comment:
```suggestion
Base.one(::Union{T,Type{T}}) where {T<:StringView} = typemin(T)
```
since `one` should accept either a type or an instance